_slides/10-vkanithi.md

```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+Let's have a coffee break
```

Comment:
```suggestion
Let's have a snack break
```

Reply:
The pic shows Coffee, so Coffee break is better

Reply:
That is fine.  I just don't drink coffee.

Reply:
Sounds good